### PR TITLE
feat: display provider response for technical failures

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Display the provider response on the dashboard if it's available. This is only set for technical failures.

Screen Shot 2021-04-14 at 11 51 17
Screen Shot 2021-04-14 at 11 51 24

See https://github.com/cds-snc/notification-api/pull/1265 for the API pull-request. ⚠️ The API pull-request will need to go out first.

Trello: https://trello.com/c/IUiDX5bk/443-communicate-sms-opt-outs-to-services-via-dashboard-error

### CODE REVIEW

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/66da02a104b9454bb7d0de53d55c69622f7759f4

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/bfb748d2c0611510754e233a3fb6466099b26a06

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/ed065a54227d705e4234a5fed52f829866355ecd

File: app/translations/csv/fr.csv [hunk 1]
```diff
@@ -1325,3 +1325,13 @@
 "Benefits of GC Notify","Avantages de GC Notify"
 "Ease of use","Facilité d'utilisation"
 "Counter","Compteur"
+"Blocked as spam by phone carrier","Bloqué comme spam par le fournisseur téléphonique"
```
Comment (AntoineAugusti) [inline app/translations/csv/fr.csv:1328]: Reasons are coming from https://docs.aws.amazon.com/sns/latest/dg/sms_stats_cloudwatch.html#sms_stats_delivery_fail_reasons

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/a2d8059c124fbc6e53c2ad2d3c8d770e55de37cb

**Comment by AntoineAugusti** [comment]
@QuanNguyen-CDS @anikbrazeau The API PR went out to production, this can go out when you want

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/71215ab926e956e10ab5c214ae9f50eee9a3d74f

**Review by QuanNguyen-CDS — APPROVED** [review]
LGTM

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/1b2f6156af9491868be7fccea1357f2949fd525b

**Comment by maxneuvians** [comment]
https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fnotification/d1df7761e896ef182375b0e5f903bfe33ba8cca5

**Pull request merged**